index.bs

```diff
@@ -475,7 +475,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
     1. If |extension| is not supported by this client platform, then [=continue=].
 
     1. Otherwise, let |result| be the result of running |extension|'s [=client processing=] algorithm on |argument|. If the
-        algorithm returned an error, [=continue=].
+        algorithm returned an error, return [=a promise rejected with=] a {{DOMException}} whose name is
+        "{{TypeError}}", and terminate this algorithm.
```

Comment:
this adds signaling of errors from running |extension|'s [=client processing=] algorithm, and aborting of the entire makeCred() operation, where the prior behavior (whether actually good or not) is to ignore errors and go on to the next extension and not abort the overall operation.

Others e.g. @vijaybh @jcjones @jyasskin ought to weigh in on this algorithm change.